src/Microsoft.Azure.WebJobs.Host/Bindings/AttributeCloner.cs

```diff
@@ -305,47 +304,44 @@ internal TAttribute ResolveFromBindings(IReadOnlyDictionary<string, object> bind
             return newAttr;
         }
 
-        // Resolution policy for { } in  binding templates. 
-        // The default policy is just a direct substitution for the binding data. 
-        // Derived policies can enforce formatting / escaping when they do injection. 
-        private class ResolutionPolicy
+        internal static IResolutionPolicy GetPolicy(PropertyInfo propInfo)
         {
-            public string TemplateBind(BindingTemplate template, IReadOnlyDictionary<string, object> bindingData)
-            {
-                if (bindingData == null)
-                {
-                    return template.Pattern;
-                }
-                return this.TemplateBindWorker(template, bindingData);
-            }
+            AutoResolveAttribute autoResolveAttribute = propInfo.GetCustomAttribute<AutoResolveAttribute>();
 
-            protected virtual string TemplateBindWorker(BindingTemplate template, IReadOnlyDictionary<string, object> bindingData)
-            {                
-                return template.Bind(bindingData);
-            }
+            var formatterType = autoResolveAttribute.ResolutionPolicyType;
 
-            // Expand on [AutoResolve] property to make this pluggable, 
-            public static ResolutionPolicy GetPolicy(PropertyInfo propInfo)
+            if (formatterType != null)
             {
-                if (propInfo.DeclaringType == typeof(TableAttribute) && propInfo.Name == "Filter")
+                // Special-case Table as there is no way to declare this ResolutionPolicy
+                // and use BindingTemplate in the Core assembly
+                if (formatterType.FullName == typeof(ODataFilterResolutionPolicy).FullName)
```

Comment:
>FullName [](start = 34, length = 8)

do we need the .FullName? 
just do :
  (formatterType == typeof(ODataFilterResolutionPolicy))